ast/desugar/Desugar.cc

```diff
@@ -248,9 +248,11 @@ unique_ptr<Expression> unsupportedNode(DesugarContext dctx, parser::Node *node)
 }
 
 unique_ptr<Expression> desugarMlhs(DesugarContext dctx, core::Loc loc, parser::Mlhs *lhs, unique_ptr<Expression> rhs) {
-    InsSeq::STATS_store stats;
+    InsSeq::STATS_store stats(2); // we fill these holes at the end
```

Comment:
could you remove `(2)` and use `emplace_back` instead?
We in general try not to do micro-optimizations at cost of safety before profiler tells us to.

Reply:
It had to be done with insert since it's putting items in the front of the vector. I also had to insert in reverse order than what ends up in the vector. Done in dad913f